src/src/WebServer/Markup.cpp

```diff
@@ -769,7 +769,7 @@ void addTextBox(const String  & id,
 {
   addHtml(F("<input "));
   addHtmlAttribute(F("class"),     classname);
-  addHtmlAttribute(F("type"),      F("search"));
+  addHtmlAttribute(F("type"),      F("text"));
```

Comment:
Just curious, what is the difference between these types?

Reply:
`search` gives you an x at the end of the field to clear the field and with "esc" you can also clear it.
`text` just a plain textfield

the reason i change it was that the x was blocking the field when viewed on a mobile:
`search`:

<img width="332" alt="Bildschirmfoto 2022-12-01 um 10 50 48" src="https://user-images.githubusercontent.com/33860956/205022075-b94b9747-5416-41da-abae-998a0a2250b9.png">
<img width="150" alt="Bildschirmfoto 2022-12-01 um 10 51 09" src="https://user-images.githubusercontent.com/33860956/205022083-76b5b599-76ef-4f0b-9dd8-90218f34bc83.png">

`text`:

<img width="343" alt="Bildschirmfoto 2022-12-01 um 10 51 54" src="https://user-images.githubusercontent.com/33860956/205022268-6b744c9b-7b34-4424-99aa-ca66ed9c1e70.png">
<img width="135" alt="Bildschirmfoto 2022-12-01 um 10 52 26" src="https://user-images.githubusercontent.com/33860956/205022274-ab07a17d-b01b-45e9-a0e2-fb488efda02b.png">

Reply:
The reason I added it, was to make it easier to clear the content of a field.
Especially when adding lots of nrs in fields, like the calibration fields on the ADC plugin, but also the task value names will be filled in with the default names when being "empty". This makes it easier to use them when changing output types.

So when it is blocking, maybe it should be made bigger?

Reply:
I can change that back... i actually never used the "x" nor the "esc"-key to clear a field so i didn´t think about that 🙂

> So when it is blocking, maybe it should be made bigger?
No its fine... it just a minor issue that for me is totally fine to live without it. And when the masses will ever start to complain we can the still do something about it. 😁